src/schema/nmdc.yaml

```diff
@@ -2671,6 +2755,62 @@ slots:
     range: PersonValue
     description: Principal Investigator who led the study and/or generated the dataset.
 
+  associated_dois:
+    description: A list of DOIs associated with a resource, such as a list of DOIS associated with a Study.
+    aliases:
+      - Associated DOIs
+      - Associated digital object identifiers
+    range: Doi
+    multivalued: true
+    in_subset: data_portal_subset
+    examples:
+      value:
+      - doi: doi:10.46936/intm.proj.2021.60141/60000423
+        doi_provider: emsl
+        doi_category: award_doi
+      - doi: doi:10.1101/2022.12.12.520098
+        doi_category: publication_doi
+      description: provides a list of two DOIs, an EMSL award DOI and a publication DOI.
```

Comment:
Also, when I first read the sentence, I thought it was a list of three things (two DOIs, an EMLS award DOI, and a publication DOI). I recommend replacing the comma with something like:

```diff
- a list of two DOIs, an EMSL award DOI and a publication DOI.
+ a list of two DOIs; specifically, an EMSL award DOI and a publication DOI.
```
> I'm assuming using a `:` would break YAML parsing (unless the whole string is enclosed in quotes)

```diff
- ...a list of two DOIs, an EMSL award DOI and a publication DOI.
+ "...a list of two DOIs: an EMSL award DOI and a publication DOI."
```

In case a `:` isn't an option, I think an "[em dash](https://www.thepunctuationguide.com/em-dash.html)" (`—`, which is `option + shift + -` on a Mac) would work well:
```diff
- a list of two DOIs, an EMSL award DOI and a publication DOI.
+ a list of two DOIs—an EMSL award DOI and a publication DOI.
```